app/views/account.py

```diff
@@ -52,6 +53,7 @@ def post(self):
                 account_number=account_number,
                 is_active=1,
                 is_deleted=0,
+                account_balance=1000,
```

Comment:
Why `account_balance` is set to 1000 always?

Reply:
when we creating account by default it get initialies to 1000
like minimum amount credit while creating the account
so account_balance is set to 1000